### PR TITLE
breaks stats into attrs and fixes line parse

### PR DESCRIPTION
**Pull request by joooooooooe-star**
**Purpose**
1: Player event parse was not correct
2: Breaks stats into attributes

**New Features**
None

**Bug Fixes**
Mentioned by Luna in the Discord

**Before/After**
Waiting for tests to make sure it works

**Additional Context**
I am 100% sure this is ugly is sin and I'm hoping I'm on the right track and someone can provide some sort of feedback to make it look prettier

### CODE REVIEW

File: nari/io/reader/actlogutils/playerstats.py [hunk 1]
```diff
@@ -31,12 +31,35 @@ def playerstats_from_logline(timestamp: datetime, params: List[str]) -> PlayerSt
...
+
+    param_ints = [int(param) for param in params[0:param_count] if param != "0"]
     # Malformed line check
     if len(param_ints) != 16:
         raise ActLineReadError("Params are unexpectedly short")
 
-    job = Job(param_ints.pop(0))
-    stats = [Stats(param) for param in param_ints]
+    job = Job(param_ints[0])
+    stats: Dict[Stats, int] = {param_to_stat[i]: param for i, param in enumerate(param_ints)}
```
Comment (lunarrize) [inline nari/io/reader/actlogutils/playerstats.py:63]: why not do this

```python
stat_order: List[Stats] = [Stats.STR, Stats.DEX, Stats.VIT, ...]
stats: Dict[Stats, int] = {stat: v for stat,v in zip(stat_order, param_ints)}
```

**Review by lunarrize — CHANGES_REQUESTED** [review]

File: nari/io/reader/actlogutils/playerstats.py (hunk 1 above)
Comment (joooooooooe-star) [reply nari/io/reader/actlogutils/playerstats.py]: fixed

File: nari/io/reader/actlogutils/playerstats.py [hunk 2]
```diff
@@ -10,33 +10,37 @@
...
-    7: ATTACK POWER
-    8: DHIT
-    9: CRIT
-    10: ATTACK MAGIC POTENCY
-    11: HEAL MAGIC POTENCY
-    12: DET
-    13: SKILL SPEED
-    14: SPELL SPEED
-    15: Blank (0)
-    16: TENACITY
```
Comment (lunarrize) [inline nari/io/reader/actlogutils/playerstats.py:31]: why were these lines removed? they document the params

**Review by lunarrize — CHANGES_REQUESTED** [review]
looks good just add docs back

**Review by lunarrize — APPROVED** [review]
thanks

**Pull request merged**